include/onnxruntime/core/providers/nnapi/nnapi_provider_factory.h

```diff
@@ -19,9 +19,15 @@ enum NNAPIFlags {
   // Please note for now, NNAPI perform worse using NCHW compare to using NHWC
   NNAPI_FLAG_USE_NCHW = 0x002,
 
+  // Disable using NNAPI CPU implementation (nnapi-reference) for execution
+  // Please note, by enabling this option, it may cause the model cannot be executed using
```

Comment:
Possibly not clear enough given there's NNAPI, the NNAPI EP and our CPU implementations. Is '(nnapi-reference)' meant to be a link to some NNAPI doco?

Maybe something a bit more verbose/explicit like
>Prevent NNAPI from using CPU devices. If NNAPI would use a CPU device to execute a node, and this flag is set, use the ORT kernel instead.

Also need a comment explaining why someone would set this.